src/components/AdminDashboard/Incidents.js

```diff
@@ -1,123 +1,23 @@
 import React from 'react';
 
-import PendingIncident from './PendingIncident';
-import AdminTable from './AdminTable';
 import AntTable from './AntTableComponents/AntTable';
-import Incidents from '../incidents/Incidents';
 
 const DashboardIncidents = props => {
   const {
-    confirmApprove,
-    confirmReject,
-    confirmApproveHandler,
-    confirmRejectHandler,
-    approveAndRejectHandler,
-    confirmCancel,
     selected,
-    selectAll,
-    allSelected,
-    handlePerPageChange,
-    currentSet,
-    setUnapprovedIncidents,
-    setPageNumber,
-    unapprovedIncidents,
     setSelected,
-    formResponses,
```

Comment:
This is a lot of change, going to have to see the loom video before considering approval.

Reply:
I touch on this in the video. The callbacks and state being passed in is mostly for dealing with the Approve/Reject buttons. Those buttons have been pulled out into their own component, and Incidents.js sole purpose is to show a list of selectable incidents.

Reply:
Although probably this component is not even necessary: AntTable could be used directly from AdminDashboard since this doesn't actually do anything but pass props a level deeper.